app/src/main/res/values/strings.xml

```diff
@@ -179,6 +179,9 @@
     <string name="preference_tts_message_format_key" translatable="false">tts_message_format_key</string>
     <string name="preference_tts_message_format_title">Message format mode</string>
     <string name="preference_tts_header">TTS</string>
+    <string name="checkered_messages_key">checkered_messages</string>
```

Comment:
Make sure to make this non-translatable
```suggestion
    <string name="checkered_messages_key" translatable="false">checkered_messages</string>
```

Reply:
Fixed in 28aa7021e7b83d10548bd5fc26c804c8a7ef64d6